app/src/utils/hooks.ts

```diff
@@ -138,6 +138,40 @@ export const usePageParams = () => {
   return { page, pageSize, setPageParams };
 };
 
+export const useVisibleEvaluationColumns = () => {
```

Comment:
This should definitely live in the `/Evaluation/EvaluationTable` directory. There's no reason to throw everything in one file just because it happens to be a hook.